src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -41,25 +45,43 @@
     <DebugTestFrameworkFolder>net45</DebugTestFrameworkFolder>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TestWithCore)' == 'true'">
-    <TestTargetFramework Include=".NETCoreApp,Version=v1.0">
+  <ItemGroup>
+    <TestNugetTargetMoniker Include=".NETCoreApp,Version=v1.0" Condition="'$(TestTFM)' == 'netcoreapp1.0'">
       <Folder>netcoreapp1.0</Folder>
-    </TestTargetFramework>
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TestWithCore)' != 'true'">
-    <TestTargetFramework Include=".NETFramework,Version=v4.6.2" Condition="'$(TargetGroup)' == 'net462'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6.3" Condition="'$(TestTFM)' == 'net463'">
+      <Folder>net462</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6.2" Condition="'$(TestTFM)' == 'net462'">
       <Folder>net462</Folder>
-    </TestTargetFramework>
-    <TestTargetFramework Include=".NETFramework,Version=v4.6.1" Condition="'$(TargetGroup)' == 'net461'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6.1" Condition="'$(TestTFM)' == 'net461'">
       <Folder>net461</Folder>
-    </TestTargetFramework>
-    <TestTargetFramework Include=".NETFramework,Version=v4.6" Condition="'$(TargetGroup)' == 'net46'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6" Condition="'$(TestTFM)' == 'net46'">
       <Folder>net46</Folder>
-    </TestTargetFramework>
-    <TestTargetFramework Include=".NETFramework,Version=v4.5" Condition="'$(TargetGroup)' == 'net45'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.5" Condition="'$(TestTFM)' == 'net45'">
       <Folder>net45</Folder>
-    </TestTargetFramework>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.5.1" Condition="'$(TestTFM)' == 'net451'">
+      <Folder>net451</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.5.2" Condition="'$(TestTFM)' == 'net452'">
+      <Folder>net451</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include="UAP,Version=v10.0" Condition="'$(TestTFM)' == 'netcore50aot'">
+      <Folder>netcore50aot</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETStandard,Version=v1.3" Condition="'$(TestTFM)' == 'netstandard13aot'">
```

Comment:
what mapping is that?  netstandard13aot is a made up string.
Moreover, I don't think it makes sense to "test" against a NETStandard TFM.  Those aren't meant to be runnable app-types.  We may compile portable test assets against netstandard, but when running them we should be using concrete app-model specific TFMs.